### ISSUE TITLE
Row: Can't tab to links in fullOverlay 

### ISSUE THREAD

**Issue by KaidenR** [pluralsight/design-system#138]
### Expected behavior
You should be able to see the focus ring when tabbing to an anchor tag inside the fullOverlay

### Actual behavior
There is no visual indicator when you've tabbed to the link

### Steps to repro

1. Set up a Row like this:
```
<Row fullOverlay={<Row.FullOverlayLink><a href="http://google.com">Google</a</Row.FullOverlayLink>} title="Some title" />
``` 
2. Try to tab to the link and notice that you don't see a focus ring
### Related packages

- row@2.3.0

### Environment

- OS - MacOS 10.12
- Browser version - Chrome v60

**Comment by jaketrent** [comment]
@KaidenR fixed in v2.3.1
